setup.py

```diff
@@ -58,7 +58,10 @@
           'cryptography==1.7.2',
       ],
       test_suite='nose.collector',
-      tests_require=['nose'],
+      tests_require=[
+          'nose',
+          'pyepub',
```

Comment:
I think pyepub should be listed in install_requires, not tests_require

Reply:
Right. I was forgetting that Travis is used as part of the functionality, not just tests. Done